cpu/native/stdio_native/Kconfig

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2021 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config MODULE_STDIO_NATIVE
```

Comment:
This already exists in https://github.com/RIOT-OS/RIOT/blob/83a433c8e76fea5515006002c4e4a00b6b44827e/sys/Kconfig.stdio#L41-L44

Reply:
According to the current practice, you could just `select MODULE_ISRPIPE`